contribute/code.rst

```diff
@@ -0,0 +1,211 @@
+Code changes
+============
+
+This page helps you get your changes to Ubuntu Touch's code merged. To help you with this, we describe the things we look for in code, commits, and changesets. We also take a look at what to expect in the code review process.
+
+Assumptions
+-----------
+
+This page does not include information about how to develop applications or changes to Ubuntu Touch. To learn how to develop Ubuntu Touch components, see the :doc:`/systemdev/index` section. To learn how to develop applications, see the :doc:`/appdev/index` or :doc:`/contribute/preinstalled-apps` sections.
+
+This page assumes you know how to write your changes, use Git to commit them, push them to GitHub or Gitlab as needed, fork a repository, and make a Pull Request (PR) on GitHub or Merge Request (MR) on GitLab. We are only covering what you need to put in your changes, commit messages, and changeset descriptions; and how to expect the review process will work.
+
+Definitions
+-----------
+
+Since we use GitHub and GitLab for our code hosting depending on the component, some terms must be defined up front for both platforms.
+
+changeset
+    A GitHub Pull Request (PR) or GitLab Merge Request (MR).
+issue report
+    A GitHub or GitLab Issue.
+
```

Comment:
Both gitlab and github use the word "issues". So this does not need explaining. But if we state "issue report", meaning no other issues or reports anywhere else, shouldn't we also name "commit" here too?

commit
           A code contribution that is pushed to either a Gitlab or Github repository.